tslint.json

```diff
@@ -38,7 +37,13 @@
       "test/runtime/gen/TestLeftRecursion/testReturnValueAndActionsAndLabels_3/TParser.ts",
       "test/runtime/gen/TestLeftRecursion/testReturnValueAndActionsAndLabels_4/TParser.ts",
       "test/runtime/gen/TestSemPredEvalParser/testPredTestedEvenWhenUnAmbig_1/TParser.ts",
-      "test/runtime/gen/TestSemPredEvalParser/testPredTestedEvenWhenUnAmbig_2/TParser.ts"
+      "test/runtime/gen/TestSemPredEvalParser/testPredTestedEvenWhenUnAmbig_2/TParser.ts",
```

Comment:
Instead of disabling these in tslint.json, lets update TypeScript.stg to emit a the following in the header:

```
/* tslint:disable:class-name */

```

See https://palantir.github.io/tslint/usage/rule-flags/

Reply:
This is a rule which is easy for users to follow by simply choosing rule names without underscores. I think it will be more important to disable some of the ones users have no control over, but first I'd like to get as close as possible to clean output.

Reply:
OK